ivy/functional/frontends/paddle/manipulation.py

```diff
@@ -44,6 +44,34 @@ def cast(x, dtype):
     return ivy.astype(x, dtype)
 
 
+@with_supported_dtypes(
+    {"2.5.1 and below": ("bool", "int32", "int64", "float16", "float32", "float64")},
+    "paddle",
+)
+@to_ivy_arrays_and_back
+def chunk(x, chunks, axis=0, name=None):
+    if x.shape == ():
+        if chunks > 1:
+            raise ValueError("Invalid value for chunks")
+        return [x]
+    axis_size = ivy.shape(x)[axis]
+    chunk_size = axis_size // chunks
+    if chunk_size == 0:
+        return ivy.split(x, num_or_size_splits=axis_size, axis=axis)
+    else:
```

Comment:
You could remove the else statement since there is a return above. Do the same for rest of the code as well.